web/src/components/GroupForm.js

```diff
@@ -117,8 +117,8 @@ const GroupForm = ({ onSubmit, initialValues = {} }) => (
                 value={values.category}
               >
                 <Option value="Sports">Sports</Option>
-                <Option value="Initiative">Initiative</Option>
-                <Option value="Arts">Arts</Option>
+                <Option value="Initiative">Outreach and Innovation</Option>
```

Comment:
The value of the option gets into firebase so even if we rename the option label to Outreach and Initiative, it's still Initiative in firebase. Change the value to "Outreach and Innovation" as well